src/applications/vaos/containers/ExpressCareInfoPage.jsx

```diff
@@ -1,137 +1,168 @@
-                Submit an Express Care request online
-              </p>
-              You can request Express Care today between{' '}
-              {localWindowString?.replace(' to ', ' and ')}. You don’t need to
-              have an assigned Patient Aligned Care Team (PACT) to use Express
-              Care.
-            </li>
-            <li className="process-step list-two">
-              <p className="vads-u-font-size--h4 vads-u-font-family--serif vads-u-padding-y--0p25 vads-u-font-weight--bold">
-                Wait for a call from VA health care staff
-              </p>
-              Someone from VA health care staff will call you soon after you
-              submit your request. This call won’t be from your primary care
-              provider or someone on your PACT team.
-            </li>
-            <li className="process-step list-three">
-              <p className="vads-u-font-size--h4 vads-u-font-family--serif vads-u-padding-y--0p25 vads-u-font-weight--bold">
-                Talk to the VA health care staff member about your health
-                concern
-              </p>
-              The VA health care staff will assess your condition and recommend
-              next steps. Depending on your needs, they might recommend you have
-              a follow-up phone or video call with your primary care provider or
-              a specialist. They might recommend a prescription. Not all Express
-              Care calls will result in a follow-up appointment.
-            </li>
-          </ol>
-        </div>
-        <AlertBox
-          headline="Express Care can’t provide emergency help"
-          status="warning"
-          className="vads-u-margin-top--0p5 vads-u-margin-bottom--4"
-          isVisible
-        >
-          <p>
-            <strong>
-              Call <a href="tel:911">911</a> or go to the nearest emergency room
-              now if you have any of these symptoms:
-            </strong>
-          </p>
-          <ul>
-            <li>Major bleeding or trauma</li>
-            <li>Chest pain with shortness of breath</li>
-            <li>Sudden inability to speak or walk</li>
-            <li>Vaginal bleeding or abdominal pain during pregnancy</li>
-            <li>Sudden, severe headache</li>
-            <li>Sudden vision loss</li>
-          </ul>
-          <p>
-            <b>Note: </b>
-            For emergencies, you don’t need a VA referral or approval to go a
-            non-VA ER in your community.
-          </p>
-          <p>
-            <strong>If you need to talk to someone right now:</strong>
-          </p>
-          <ul>
-            <li>
-              Call our Veterans Crisis Line at{' '}
-              <a href="tel:800-273-8255">800-273-8255</a> and select 1.
-            </li>
-          </ul>
-        </AlertBox>
+      <ExpressCareRequestLimitError
```

Comment:
This should arguably be a new page, instead of replacing all the contents of the current page.

Reply:
I was considering that but I gave it a second thought since I don't think we've had separate pages for error messages in the past.  But yeah, since we're replacing the entire page content, I can split this out.